### ISSUE TITLE
Proposal: Adopt Symfony coding stardards

### ISSUE THREAD

**Issue by aleksip** [pattern-lab/patternlab-php-core#150]
Pattern Lab PHP uses many Symfony components, so it might make sense to adopt their coding standards. They also have a nice tool, the <a href="http://cs.sensiolabs.org/">PHP Coding Standards Fixer</a>, which we could run on our entire PHP codebase.

**Comment by sghoweri** [comment]
^ and could probably use even more Symphony components to further reduce the amount of custom code needing to be maintained.

@aleksip at a high level, what sort of things would adopting Symphony's coding standards change? Please oh please oh please don't say we need to switch back over to using tabs 😜

**Comment by aleksip** [comment]
> and could probably use even more Symphony components to further reduce the amount of custom code needing to be maintained

Indeed! Actually, I have some secret plans related to this I'm hoping to advance next week. 😉 

> Please oh please oh please don't say we need to switch back over to using tabs

Symfony follows PSR-2 which dictates that code MUST use 4 spaces for indenting, not tabs!

**Comment by EvanLovely** [comment]
> secret plans

😳🤔

**Comment by aleksip** [comment]
Not evil ones, I promise! 😈 😄 

Just planning to find out if some custom PL code could be replaced with a drop in Symfony replacement with no side effects!
